insta-clone/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './PostContainer.css';
+import Header from './Header';
+import Body from './Body';
+import CommentSection from '../CommentSection/CommentSection';
+
+
+const PostContainer = (props) =>{
+    return(
+        <div>
+            <Header thumbnail={props.obj.thumbnailUrl} username={props.obj.username}/>
+            <Body postPic={props.obj.imageUrl} />
+            <CommentSection comArray={props.obj.comments} likes={props.obj.likes}/>
+        </div>
+    );
+};
+
```

Comment:
Try to implement a props-type check anywhere where you are passing down props data